source/RoslynAnalyzers/RoslynAnalyzers.csproj

```diff
@@ -22,20 +22,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.2" />
-    <!--
-    Using this version is intentional!
-    The newer version(s) break under the 'dotnet build' chain when this is added as a nuget package to another project. (Tested under .NET 5.0.101)
-    Fails with versions:
-      Microsoft.CodeAnalysis.CSharp 3.8.0
-      Microsoft.CodeAnalysis.CSharp 3.3.9.0-4.21056.26
-    -->
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.7.0" />
+    <None Update="tools\*.ps1" CopyToOutputDirectory="Always" Pack="true" PackagePath="" />
+    <None Visible="false" Include="$(OutputPath)\$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" />
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="tools\*.ps1" CopyToOutputDirectory="Always" Pack="true" PackagePath="" />
-    <None Visible="false" Include="$(OutputPath)\$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.3.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.3.0" />
```

Comment:
- This is required to reference `CodeFixProvider`
- Taking this opportunity to update reference to latest available